tests/unit/javascript/stylesheet/simple.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Might want to pick something simpler here - I think an XML with just one tag in it would be nicer.